app/controllers/posts_controller.rb

```diff
@@ -22,9 +22,10 @@ def create
     respond_to do |format|
       format.html do
         if @new_post.save
-          redirect_to "/users/#{@new_post.user.id}/posts/", notice: 'Success!'
+          redirect_to "/users/#{@new_post.user.id}/posts/", flash[:notice] = 'Your post is saved!'
```

Comment:
- [ ] While creating a post, I'm getting the error of `no implicit conversion of symbol into string`. Please fix this.

![Screenshot from 2022-02-25 22-56-38](https://user-images.githubusercontent.com/22434660/155760845-44380918-fbaf-4fe0-9471-453ae8cbc8b9.png)

Reply:
Thanks for your time and feedback. I'll fix it ASAP.